docs/running.rst

```diff
@@ -22,6 +22,22 @@ To run on 4 machines with 4 GPUs each:
 
     $ horovodrun -np 16 -H server1:4,server2:4,server3:4,server4:4 python train.py
 
+Host nodes can also be specified in a host file:
+For example, 
+
+$ cat myhostfile
+aa slots=2
+bb slots=2
+cc slots=2
```

Comment:
[optional] I would suggest using the same format of the --host option for the host file as well  i.e., each line could be 

$cat myhostfile
host1:np1
host2:np2
...

Reply:
This file format is defined by mpirun: https://www.open-mpi.org/faq/?category=running#mpirun-hostfile